spring-content-s3/src/test/java/internal/org/springframework/content/s3/store/DefaultS3StoreImplTest.java

```diff
@@ -375,7 +378,156 @@ public S3ObjectId convert(CustomContentId entity) {
                         });
                     });
 				});
-				Context("associate", () -> {
+				Context("#getResource with PropertyPath", () -> {
```

Comment:
The tests in this context are partially an exact copy of those above (only other function with PropertyPath is called), and the second part is similar to the recently added S3StoreWithEntityConverterIT (except there are more different false ContentPropertyInfo converters here). I can delete them if it bloats the number of similar tests.